README.md

```diff
@@ -10,92 +14,56 @@ It provides users with clean structured and label-enriched information set for r
 
 - [Parsr: Turn your documents into data!](#parsr-turn-your-documents-into-data)
   - [Getting Started](#getting-started)
-  - [Usage](#usage)
+    - [Installation](#installation)
+    - [Usage](#usage)
   - [Documentation](#documentation)
   - [Contribute](#contribute)
   - [Third Party Licenses](#third-party-licenses)
   - [License](#license)
 
 ## Getting Started
 
-*-- The advanced installation guide is available [here](docs/installation.md) --*
-
-The quickest way to install Parsr is using the [docker installation](docs/docker.md#1-run-parsr).
+### Installation
 
-Otherwise, Parsr can also be installed manually.
-
-First, install the dependencies:
+*-- The advanced installation guide is available [here](docs/installation.md) --*
 
-**Under Linux: (Ubuntu/Debian)**
+The quickest way to install and run the Parsr API is through the [docker image](https://hub.docker.com/r/axarev/parsr):
 
 ```sh
-sudo add-apt-repository ppa:ubuntuhandbook1/apps
-sudo apt-get update
-sudo apt-get install nodejs npm qpdf imagemagick graphicsmagick python-pdfminer tesseract-ocr libtesseract-dev python3-tk ghostscript python3-pip
-pip install camelot-py
+docker pull axarev/parsr
 ```
 
-**Under MacOS (requires [Homebrew](http://brew.sh))**
-
+If you also wish to install the GUI for sending documents and visualising results:
 ```sh
-brew install node qpdf imagemagick graphicsmagick tesseract tesseract-lang
-curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
-python get-pip.py
-pip install pdfminer.six
-pip install ghostscript camelot-py
+docker pull axarev/parsr-ui-localhost
 ```
 
-**Under Windows**
+Note: Parsr can also be installed bare-metal (not via Docker containers), the procedure for which is documented in the [installation guide](docs/installation.md).
 
-See [our guide](docs/installation.md#13-windows) for the step by step Windows installation procedure.
-
-
-## Usage
+### Usage
 
 *-- The advanced usage guide is available [here](docs/usage.md) --*
 
-1. First, clone the respository:
-    ```sh
-    git clone git@github.com:axa-group/Parsr.git
-    ```
-
-2. Then, install the npm packages:
-    ```sh
-    cd Parsr
-    npm install
-    ```
-
-3. Launch Parsr's GUI tool:
-
-    Under **MacOS** and **Linux**:
-
-    ```sh
-    npm run start:web:vue
-    ```
-    Under **Windows**:
-
-    In one terminal:
-
-    ```sh
-    npm run start:api
-    ```
-
-    In another terminal:
-
-    ```sh
-    cd demo/vue-viewer && npm install && npm run serve
-    ```
+To run the [API](docs/api-guide.md), issue:
+```sh
+docker run -p 3001:3001 axarev/parsr
+```
+which will launch it on [http://localhost:3001](http://localhost:3001).  
+Consult the documentation on the [usage of the API](docs/api-guide.md).
 
-    Then, access [http://localhost:8080](http://localhost:8080).
+To use the GUI tool (the API needs to already be running), issue:
+```sh
+docker run -p 3001:3001 axarev/parsr
```

Comment:
I think this should be `docker run -p 8080:8080 axarev/parser-ui-localhost`

Reply:
pars**E**r --> parsr --> `docker run -p 8080:8080 axarev/parsr-ui-localhost`

Reply:
I updated docker command to `docker run -p 8080:8080 axarev/parsr-ui-localhost`

But I'm still waiting DevOps feedback....